src/languages/fsharp.js

```diff
@@ -127,20 +123,47 @@ export default function(hljs) {
         ]
       },
       {
-        className: 'meta',
-        begin: '\\[<',
-        end: '>\\]',
-        relevance: 10
+        // computation expressions:
+        beginScope: "emphasis",
+        match: /\b[_a-z]\w*(?=\s*\{)/i
+      },
+      {
+        // preprocessor directives and fsi commands:
+        scope: 'meta',
+        begin: '#',
+        end: '$',
+        keywords: {
+          keyword: 'if else endif line nowarn light r i I load time help quit'
+        }
+      },
+      {
+        // [<Attributes("")>]
+        scope: 'meta',
+        begin: /^\s*\[<(?=[<\w])/,
+        excludeBegin: true,
+        end: />\]/,
+        excludeEnd: true,
+        relevance: 10,
```

Comment:
Perhaps 2, we don't really hand out 10s anymore unless we're sure no other language could ever contain such a thing.